SharedPackages/BrowserServicesKit/Sources/RemoteMessaging/Mappers/JsonToRemoteMessageModelMapper.swift

```diff
@@ -244,6 +250,8 @@ struct JsonToRemoteMessageModelMapper {
             return .newForMacAndWindows
         case .privacyShield:
             return .privacyShield
+        case .duckAi:
```

Comment:
Internally we've been referring to duck.ai as AI Chat for consistency. I think it would be good to keep doing it so we don't have both AIChat and DuckAI